index.html

```diff
@@ -212,14 +212,19 @@ <h2 class="form-heading">Contact me</h2>
     <p class="form-content">If you have an application you are interested in developing, a feature that you need built
       or a project that needs coding. I’d love to help with it.</p>
     <form class="contact-form" action="https://formspree.io/f/mvolvljr" method="POST">
-      <input class="name" type="text" name="firstname" maxlength="30" placeholder="Please enter your name " required>
-      <input class="email" type="email" name="email" placeholder="Please enter your email" required>
-      <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
-        required></textarea>
-      <button class="submit" type="submit">Get it touch</button>
+     <ul>
+      <li><input class="name" type="text" name="firstname" maxlength="30" placeholder="Please enter your name " required></li>
+      <li><input class="email" type="email" name="email" placeholder="Please enter your email" required></li>
+      <li>
+        <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
+          required></textarea></li>
+          <li><button class="submit" type="submit">Get it touch</button></li>
+     </ul>
     </form>
```

Comment:
- [ ] Please add the background images for the contact section and make sure it matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/187513420-d07654b1-7fbb-48b0-b839-322c50cf6a74.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/187513384-4a5fe52d-54a4-4140-822a-ab4475eac418.png)